gradle-conjure-api/src/main/java/com/palantir/gradle/conjure/api/ConjureExtension.java

```diff
@@ -31,12 +32,16 @@ public class ConjureExtension {
     private final GeneratorOptions pythonOptions = new GeneratorOptions();
     private final Map<String, GeneratorOptions> genericOptions = new HashMap<>();
 
-    public ConjureExtension() {
+    public ConjureExtension(Project project) {
```

Comment:
we could potentially change this to be `String jakartaPackagesProperty` and just make the caller use `project.findProperty` - i'm not really sure which is preferred.